library/psa_crypto.c

```diff
@@ -235,6 +237,17 @@ psa_status_t mbedtls_to_psa_error( int ret )
         case MBEDTLS_ERR_GCM_HW_ACCEL_FAILED:
             return( PSA_ERROR_HARDWARE_FAILURE );
 
+#if !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG) &&        \
+    defined(MBEDTLS_PSA_HMAC_DRBG_MD_TYPE)
```

Comment:
I think is clearer to use `MBEDTLS_HMAC_DRBG_C` here. These error code should only be available when the HMAC_DRBG module is enabled.
```suggestion
#if !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG) &&        \
    defined(MBEDTLS_PSA_HMAC_DRBG_C)
```